packages/stack/src/views/Stack/Card.tsx

```diff
@@ -36,7 +36,7 @@ type Props = ViewProps & {
   closing: boolean;
   next?: Animated.AnimatedInterpolation<number>;
   current: Animated.AnimatedInterpolation<number>;
-  gesture: Animated.Value;
+  gesture?: Animated.Value;
```

Comment:
if the gesture is `undefined`, the route is preloaded. Hence, we use that logic to detect when the animation shall start.